Assignment/Program.cs

```diff
@@ -1,7 +1,15 @@
-﻿namespace Assignment {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
-      }
+﻿Console.Write ("Enter text : ");
+string word = Console.ReadLine ();
+string str = word.ToLower ().Replace (" ", "");
+bool isogram = Isogram (str);
+if (isogram) Console.WriteLine ($"The given string {word} is an Isogram");
+else Console.WriteLine ($"The given string {word} is not an isogram");
+bool Isogram (string str) {
+   char[] s = str.ToCharArray ();
+   Array.Sort (s);
+   for (int i = 0; i < s.Length - 1; i++) {
+      if (s[i] == s[i + 1])

```

Comment:
Try to validate i+1 falls within string length.

Reply:
I think this validates within string length.